addons/mail/static/src/core/common/core.dark.scss

```diff
@@ -2,6 +2,10 @@ a.o_mail_redirect, a.o_channel_redirect {
     @include button-variant(rgba($primary, .2), rgba($primary, .2), lighten($link-color, 5%), rgba($primary, .3), rgba($primary, .3), lighten($link-color, 10%));
 }
 
+a.o-discuss-mention {
+    @include button-variant(rgba($primary, .2), rgba($primary, .2), lighten($link-color, 5%));
+}
```

Comment:
Should also have same hover color as non-hover
_(there's still a bit of the btn hover style, but I think it's reduced with this change)_